example/cr-detailed.yaml

```diff
@@ -1,85 +1,83 @@
 apiVersion: "pravega.pravega.io/v1beta1"
 kind: "PravegaCluster"
 metadata:
-  name: "example"
+  name: pravega
+  labels:
+    app.kubernetes.io/name: "pravega-cluster"
 spec:
   version: 0.7.0
-  zookeeperUri: zk-client:2181
-
-  # Security configurations for Pravega
-  # See https://github.com/pravega/pravega/blob/master/documentation/src/docs/security/pravega-security-configurations.md
-  tls:
-    static:
-      controllerSecret: "controller-pki"
-      segmentStoreSecret: "segmentstore-pki"
-
-  bookkeeperUri: "pravega-bk-bookie-0.pravega-bk-bookie-headless.default.svc.cluster.local:3181,pravega-bk-bookie-1.pravega-bk-bookie-headless.default.svc.cluster.local:3181,pravega-bk-bookie-2.pravega-bk-bookie-headless.default.svc.cluster.local:3181"
-
+  zookeeperUri: pravega-zk-client:2181
+  externalAccess:
+    enabled: false
+    type: LoadBalancer
+  bookkeeperUri: pravega-bk-bookie-headless:3181
   pravega:
     image:
       repository: pravega/pravega
-
     controllerReplicas: 1
     controllerResources:
       requests:
-        memory: "1Gi"
+        memory: "2Gi"
         cpu: "1000m"
       limits:
         memory: "3Gi"
         cpu: "2000m"
-
     segmentStoreReplicas: 3
     segmentStoreResources:
       requests:
-        memory: "4Gi"
-        cpu: "1000m"
-      limits:
         memory: "4Gi"
         cpu: "2000m"
-
-    # Turn on Pravega Debug Logging
+      limits:
+        memory: "8Gi"
+        cpu: "4000m"
     debugLogging: false
-
+    cacheVolumeClaimTemplate:
```

Comment:
cacheVolumeClaimTemplate is optional for 0.7.0

Reply:
I have made cache volume template as optional